### PR TITLE
Include the name of the file if it fails to upload

### PR DESCRIPTION
**Pull request by awead**
Minor issue, but it would fix an issue in another project. As far as I can tell, there's no corresponding change required in Sufia 7 or in Curation Concerns.

### CODE REVIEW

File: spec/controllers/generic_files_controller_spec.rb [hunk 1]
```diff
@@ -134,6 +134,7 @@
           file = fixture_file_upload('/world.png', 'image/png')
           xhr :post, :create, files: [file], Filename: "The world", batch_id: "sample_batch_id", permission: { "group" => { "public" => "read" } }, terms_of_service: "1"
           expect(response.body).to include("Error occurred while creating generic file.")
+          expect(response.body).to include("world.png")
```
Comment (mjgiarlo) [inline spec/controllers/generic_files_controller_spec.rb]: Why not tweak the prior line to include `world.png`?

**Comment by mjgiarlo** [comment]
Here's where the corresponding change would go in CurationConcerns:

https://github.com/projecthydra-labs/curation_concerns/blob/5527443e85bfaa115de4cd81d028e75124d62795/app/controllers/concerns/curation_concerns/file_sets_controller_behavior.rb#L210

If you don't have cycles to submit a CC PR, could you create an issue and link to this PR? Thx!

**Comment by awead** [comment]
@mjgiarlo amended and reported: projecthydra-labs/curation_concerns#582

**Comment by mjgiarlo** [comment]
Happy to merge when Travis is :green_heart:

**Comment by awead** [comment]
@mjgiarlo ready to go.

**Pull request merged**